tests/tests/classes/Plugin.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+/**
+ * Test: Plugin Function Class
+ */
+
+const expect = require('chai').expect;
+const Plugin = require('../../../lib/classes/Plugin')({});
+
+const SPlugin = new Plugin();
+
+describe('Plugin class', () => {
+
+  after((done) => {
+    done();
+  });
+
+  it('should have a name', () => {
+    const name = SPlugin.getName();
+
+    expect(name.length).to.be.at.least(1);
+  });
+
+  it('should register actions', () => {
```

Comment:
Please add clearer description what this test is doing. Not sure I fully understand what this test is doing. Same at the hooks test velow